src/containers/Staff/staffItems.json

```diff
@@ -17,12 +17,7 @@
       "title": "Richard Hoffman - Financial Administrator",
```

Comment:
@kathleenahall I believe there are links at the top of this page as well for Valerie, please delete those too, thanks